### ISSUE TITLE
需要什么样的硬件配置？

### ISSUE THREAD

**Issue by pdsujnow** [thumt/THUMT#34]
我两块tesla K40M ，显存12G，一运行就卡死
多次都是到这个状态：
`INFO:tensorflow:seq2seq/softmax/matrix                                                                  shape    (1000, 878534)
INFO:tensorflow:seq2seq/source_embedding/bias                                                           shape    (1000,)
INFO:tensorflow:seq2seq/source_embedding/embedding                                                      shape    (1804778, 1000)
INFO:tensorflow:seq2seq/target_embedding/bias                                                           shape    (1000,)
INFO:tensorflow:seq2seq/target_embedding/embedding                                                      shape    (878534, 1000)
INFO:tensorflow:Total trainable variables size: 3626758534
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Create EvaluationHook.
INFO:tensorflow:Graph was finalized.
2018-05-30 10:42:22.524117: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA`
前几次系统自动kill进程，最后一次直接卡住不动了

**Comment by Glaceon31** [comment]
你好，12G完全够用了。从embedding矩阵的大小来看，应该是词表太大导致的，请尝试在build_vocab的时候限制词表的大小为3万或5万。

**Comment by pdsujnow** [comment]
谢谢回复，我限制词表再试一试
